### PR TITLE
feat(compe): implement nvim-compe integration

### PR DESCRIPTION
**Pull request by rainerborene**
It works amazingly well. By the way, thanks for creating `tmux-complete.vim` plugin. :beer:

### CODE REVIEW

File: lua/compe_tmux/init.lua [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+end
+
+function Source.determine(_, context)
+  return compe.helper.determine(context)
+end
+
+function Source.complete(_, context)
+  local items = vim.fn["tmuxcomplete#complete"](0, context.input)
+
+  context.callback({ items = items })
```
Comment (kabouzeid) [inline lua/compe_tmux/init.lua:19]: Replace with
```lua
    vim.fn["tmuxcomplete#async#gather_candidates"](function (items)
      context.callback({ items = items })
    end)
```
for async completion.

**Review by karimknaebel — CHANGES_REQUESTED** [review]

File: lua/compe_tmux/init.lua (hunk 1 above)
Comment (rainerborene) [reply lua/compe_tmux/init.lua]: Done. Thanks!

**Comment by kabouzeid** [comment]
Hi @wellle, could you merge this? It's a small addition. I reviewed and tested the code, everything works as intended :)

**Comment by wellle** [comment]
Hey thank you @rainerborene and @kabouzeid for this. And sorry for the delay!

**Pull request merged**